cli/src/main/java/org/jboss/as/cli/embedded/StopEmbeddedHostControllerHandler.java

```diff
@@ -52,7 +52,7 @@ public boolean isAvailable(CommandContext ctx) {
     @Override
     protected void doHandle(CommandContext ctx) throws CommandLineException {
         EmbeddedServerLaunch hostControllerLaunch = hostControllerReference.get();
-        if (hostControllerLaunch != null) {
+        if (hostControllerLaunch != null && hostControllerLaunch.getHostController() != null) {
```

Comment:
Shouldn't this be in isAvailable()? That's what will prevent the CLI offering the option.

Having it here too is fine, but people shouldn't see the command if it's not valid.